pylabnet/scripts/pulsemaster/pulsemaster.py

```diff
@@ -547,7 +549,7 @@ def prep_plotdata(self, pb_obj):
                     t1, t2 = new_t1, new_t2
 
                     # Draw the current pulse at high grid density
-                    t_ar = np.linspace(t1, t2, 2000)
+                    t_ar = np.linspace(t1, t2, 800)
```

Comment:
We probably should make the grid density a parameter in the config dict (with a default value), such that we can easily change it if needed.

Reply:
Updated in latest commit.